bin/qtile

```diff
@@ -56,13 +59,13 @@ def make_qtile():
     from argparse import ArgumentParser
     parser = ArgumentParser(
         description='A full-featured, pure-Python tiling window manager.',
-        prog='qtile')
-
+        prog='qtile',
+        )
```

Comment:
i use that pattern to be able to easier

 - sort parameters

 - add more parameters or comment them

without struggling over missing commas or brackets.